configs/puerto-rico.nrel-op.json

```diff
@@ -0,0 +1,48 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://puerto-rico-openpath.nrel.gov/api/",
```

Comment:
@shankari can you please confirm which `connectUrl` we want for Puerto Rico? Thanks!

Reply:
the base URL should be https://uprm-nicr-openpath.nrel.gov

Reply:
fixed